doc/index.qdoc

```diff
@@ -0,0 +1,79 @@
+/****************************************************************************************
+**
+** Copyright (C) 2021 Open Mobile Platform LLC
+** All rights reserved.
+**
+****************************************************************************************/
+
+/*!
+\page index.html
+\title Sailfish OS WebView Component
+
+The Sailfish OS WebView Component provides a straightforward and flexible way
+for you to introduce access to Web pages and websites into your applciation.
+
+The WebView component utilises the Sailfish OS browser engine, built using
+Mozilla gecko.
+
+\section1 Using the Sailfish OS WebView Component
+
+Currently you have to explicitly link against qtmozembed-qt5 at build time, to
+avoid a symbol issue which leads to a crash:
+
+\code
+CONFIG += link_pkgconfig
+PKGCONFIG += qt5embedwidget
+\endcode
+
+\section2 C++ API
+
+The C++ API consists of the following classes:
+
+\list
+\li the \l{SailfishOS::WebEngine}{WebEngine} class
+\li the \l{SailfishOS::WebEngineSettings}{WebEngineSettings} class
+\li the \l{SailfishOS::WebEngineUtils::DownloadHelper}{DownloadHelper} class
+\endlist
+
+\section2 QML API
+
+The QML API is provided through the Sailfish.WebView import namespace
+
+The QML types exposed in the QML API are as follows:
+
+\list
+\li \l{WebView}
+\li \l{WebViewFlickable}
+\li \l{WebViewPage}
+\li \l{RawWebView}
+\endlist
+
+\section1 Example
+
+The following examples demonstrates how the WebView component can be integrated into an existing application's QML code.
+
+Example:
+
+\code
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.WebView 1.0
+
+ApplicationWindow {
+    id: root
+    initialPage: Component {
+        Page {
+            id: page
+            WebView {
+                anchors.fill: parent
+                active: true
```

Comment:
Wondering should we revisit default value state for the WebView i.e. active: true by default.

Reply:
That would seem to make sense.

Reply:
Please file a task under JB#53778 and let's fix the default value for sailfish-components-webview side.

Reply:
Filed as JB#54001.

Reply:
Thanks